clcache.py

```diff
@@ -85,17 +85,26 @@
 Manifest = namedtuple('Manifest', ['includeFiles', 'hashes'])
 
 
-def printBinary(rawData, stream):
-    # See http://stackoverflow.com/questions/2374427/python-2-x-write-binary-output-to-stdout
-    if sys.version_info[0] < 3:
-        # msvcrt and O_BINARY are only available on Windows, but we like to
-        # use pylint on Unix developer machines to speed-up development, thus
-        # we disable some lint error on a per-line base here.
-        import msvcrt # pylint: disable=import-error
-        msvcrt.setmode(stream.fileno(), os.O_BINARY) # pylint: disable=no-member
+class BinaryPrinter(object):
```

Comment:
Interesting - do you think that this has something to do with the Python 2.7-specific (possibly Unicode-related) issues with launching that peculiar integration test?